starknet_devnet/starknet_wrapper.py

```diff
@@ -416,7 +418,7 @@ def _update_block_number(self):
         )
         return block_info
 
-    def __get_transaction_handler(self, external_tx=None):
+    def __get_transaction_handler(self, external_tx=None, context: LogContext = None):
```

Comment:
Here and in other places, instead of passing the context object around, couldn't we modify a global object?
[Related SO question](https://stackoverflow.com/questions/50556584/should-logger-be-an-argument-or-a-global-variable)

Reply:
I had the thought that someday the app would be used asynchronously, making it difficult to comprehend the sequence of log entries. To address this, I created a context that keeps track of a single transaction and outputs information when it is fully completed. 
It seems I'm wrong, I will rewrite it to log the progress at each stage.